lib/srv/db/elasticsearch/engine.go

```diff
@@ -118,6 +118,10 @@ func (e *Engine) HandleConnection(ctx context.Context, sessionCtx *common.Sessio
 
 	clientConnReader := bufio.NewReader(e.clientConn)
 
+	if sessionCtx.Identity.RouteToDatabase.Username == "" {
+		return trace.BadParameter("database username required")
```

Comment:
Is it worth mentioning elasticsearch specifically in the error message? This way we don't lead the user to think the username is always required.

Reply:
Sure, we can have that.